lib/fiddle/pack.rb

```diff
@@ -48,7 +48,7 @@ module PackInfo # :nodoc: all
       ALIGN_MAP[TYPE_LONG_LONG] = ALIGN_MAP[-TYPE_LONG_LONG] = ALIGN_LONG_LONG
       PACK_MAP[TYPE_LONG_LONG] = PACK_MAP[-TYPE_LONG_LONG] = "q"
```

Comment:
Should we use `Q` for `-TYPE_LONG_LONG` too?

Reply:
Good catch, thank you https://github.com/ruby/fiddle/pull/110/commits/43b1fe4cfd95b719d8bef67f3b49d60f439cfd9e